pyquil/operator_estimation.py

```diff
@@ -819,6 +829,9 @@ def measure_observables(qc: QuantumComputer, tomo_experiment: TomographyExperime
                 # Keep track of qubit-classical register mapping via dict
                 d_qub_idx[q] = i
             total_prog_no_symm.wrap_in_numshots_loop(n_shots)
+            print ("Here's total_prog_no_symm")
```

Comment:
This looks like leftover debugging code -- should be removed.

Reply:
It's been removed.